### PR TITLE
[REVIEW] added Sagemaker, Colab and WSL2 notices

### PR DESCRIPTION
**Pull request by taureandyernv**
This PR has 

- [ ] Colab Limited Support Notice (Breaking change)
- [ ] Sagemaker Studio Labs Announcement notice (Announcement)
- [ ] WSL2 Updated Installation Method Notice  (Breaking change)

@aravenel @raydouglass , please review

### CODE REVIEW

File: _notices/rgn0022.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+notice_rapids_version: "v22.02"
+notice_created: 2022-02-17
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-02-17
+---
+
+## Overview
+
+RAPIDS `v22.02` requires Python `3.8+`; Google Colab is currently limited to
+Python `3.7`. Google Colab will only support up to RAPIDS `v22.02` until
```
Comment (aravenel) [inline _notices/rgn0022.md:30]: Is this correct? I thought support ended at 21.12.

File: _notices/rgn0022.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+layout: notice
+parent: RAPIDS General Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rgn
+# Update meta-data for notice
+notice_id: 22 # should match notice number
+notice_pin: false # set to true to pin to notice page
```
Comment (aravenel) [inline _notices/rgn0022.md:9]: I think this is probably worth pinning.

File: _notices/rgn0023.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+notice_topic: Announcement
+notice_rapids_version: "v22.02"
+notice_created: 2022-02-17
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-02-17
+---
+
+## Overview
+
+SageMaker Studio Labs has been added as a new "Try it Now" capability for all Legacy (`v21.12`), Stable (`v22.02`),
```
Comment (aravenel) [inline _notices/rgn0023.md:29]: Lets list this in order Stable, Nightly, and Legacy--no sense in leading with the old stuff!

**Review by aravenel — CHANGES_REQUESTED** [review]

File: _notices/rgn0022.md (hunk 2 above)
Comment (taureandyernv) [reply _notices/rgn0022.md:9]: d'oh.  fixed in upcoming commit

File: _notices/rgn0023.md (hunk 3 above)
Comment (taureandyernv) [reply _notices/rgn0023.md:29]: done

**Review by aravenel — APPROVED** [review]
LGTM

File: _notices/rgn0024.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+Please do not install an additional NVIDIA drivers inside WSL2.  It is no longer necessary.
+
+## Status
+
+- **12-Feb-2022** - EA support continues.
+
+## Impact
+
+Users intending to run RAPIDS `v22.02+`.
```
Comment (aravenel) [inline _notices/rgn0024.md:48]: This looks like an incomplete sentence--we should fill out the rest of the impact, e.g. "users . . .  should use the latest instructions as defined above"

**Review by aravenel — CHANGES_REQUESTED** [review]

File: _notices/rgn0024.md (hunk 4 above)
Comment (taureandyernv) [reply _notices/rgn0024.md]: addressed in commit d089b67

**Review by aravenel — APPROVED** [review]

**Pull request merged**